notebooks/en/fine_tuning_vit_custom_dataset.ipynb

```diff
@@ -0,0 +1,767 @@
+{
```

Comment:
<p>In "Dataset Info", it'd be nice to briefly explain what the images are of so users have more context about what they're training the model to do.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/85/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>